aws-ssm-patchbaseline/template.yml

```diff
@@ -4,19 +4,34 @@ Description: AWS SAM template for the AWS::SSM::PatchBaseline resource type
 
 Globals:
   Function:
-    Timeout: 60  # docker start-up times can be long for SAM CLI
+    Timeout: 30  # docker start-up times can be long for SAM CLI
+    MemorySize: 256
 
 Resources:
   TypeFunction:
     Type: AWS::Serverless::Function
     Properties:
       Handler: software.amazon.ssm.patchbaseline.HandlerWrapper::handleRequest
-      Runtime: java8
       CodeUri: ./target/aws-ssm-patchbaseline-handler-1.0-SNAPSHOT.jar
+      Runtime: java8
+      MemorySize: 512
+      Environment: # More info about Env Vars: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#environment-object
+       Variables:
+          PARAM1: VALUE
 
-  TestEntrypoint:
-    Type: AWS::Serverless::Function
+  HelloWorldFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
```

Comment:
what's the difference between this and TypeFunction above?

Reply:
TypeFunction - It will all create, update, read, list, delete handler when we do the actions on patchbaseline


HelloworldFunction - It is used to check the health check status for patchbasleine